cooltools/lib/io.py

```diff
@@ -254,7 +253,7 @@ def print_available_datasets(url=URL_DATA):
 """.format(**data))
 
 
-def read_expected(
+def read_expected_from_file(
```

Comment:
can we remove the basic input check, since this is done in `is_valid_expected()`? @sergpolly

Reply:
ph, yeah - I think it should be fine.
Assuming `if verify_view is not None:` below is skipped